src/create_disk.sh

```diff
@@ -52,13 +56,18 @@ mkfs.xfs "${disk}4"  -L root -m reflink=1
 rm -rf rootfs
 mkdir rootfs
 mount "${disk}4" rootfs
+chcon system_u:object_r:root_t:s0 rootfs
```

Comment:
Should we comment these with where we're getting these labels (i.e. what is the source of truth for these)?

Reply:
Did one better, figured out a way to avoid hardcoding.